app/ports/tests/sample_data/submissions.json

```diff
@@ -0,0 +1,206 @@
+[
+    {
+        "id": "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXX1",
+        "os": {
+            "macports_version": "2.5.4",
+            "osx_version": "10.14",
+            "os_arch": "i386",
+            "os_platform": "darwin",
+            "cxx_stdlib": "libc++",
```

Comment:
I do not get `cxx_stdlib` value on running `mpstats show` or `mpstats-gsoc show`. Is it okay to set `libc++` as default?

Reply:
Sorry, I didn't have the latest version.

Reply:
Other users who don't have the latest version may submit statistics this way. What I would do is check whether platform is `darwin` and `osx_version` is at least `10.9`, and set `libc++` in that case. If `osx_version` is smaller than that, I wouldn't dare betting about the value, and well ... we can simply leave it empty.

Maybe one of the unit tests could also cover submission from linux?

Reply:
Oh, I just realised that users who might not be having the latest version might not be able to submit posts, and the code would raise `KeyError`. I should make use of `.get`.

> Other users who don't have the latest version may submit statistics this way. What I would do is check whether platform is `darwin` and `osx_version` is at least `10.9`, and set `libc++` in that case. If `osx_version` is smaller than that, I wouldn't dare betting about the value, and well ... we can simply leave it empty.

For submissions, I did not want to set any defaults. But I am probably less knowledgeable with `stdlib` and similar configs, so I would go with this.

Reply:
Thanks for fixing the reading of data.
Yes, defaults can be left empty (except for `stdlib` in which case it's possible to know the value for 10.9+).